Utilities/bootstrap

```diff
@@ -894,7 +894,19 @@ exec ${{__dir}}/swift-$1 ${{SPM_FLAGS[@]}} "${{@:2}}" """.format(options=swiftpm
 
 def build_llbuild(args):
     note("Building llbuild")
+    
+    # Confirm ninja is available.
```

Comment:
Since we're doing the same thing in three places, we should extract this into its own method. Something like:

```
checkTool(["ninja", "--version"])
checkTool(["cmake", "--version"])
```

Reply:
done